RefApp/ViewControllers/AssetListTableViewController.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  AssetListTableViewController.swift
+//  RefApp
+//
+//  Created by Udaya Sri Senarathne on 2018-11-21.
+//  Copyright © 2018 emp. All rights reserved.
+//
+
+import UIKit
+import Exposure
+import ExposurePlayback
+
+class AssetListTableViewController: UITableViewController {
```

Comment:
The architecture implies a new viewController for other "types" of lists.
* epg
* asset types
* etc..

Reply:
I don't think its necessary to make any changes for this PR, but could be something to consider for the future

Reply:
ok